conda-env.yml

```diff
@@ -0,0 +1,14 @@
+name: dolphin-env
+channels:
+  - conda-forge
+dependencies:
+  - python>=3.7,<3.11
+  - pip
+  - gdal>=3.3
+  - git  # For pip-installing in the Docker image
```

Comment:
before I added this, when I did the `pip install .` step, it used git, so the build failed. I'll double check that I still need it with the current setup

Reply:
seems to now work without this line (though git is getting added via some other dependency now), so i'll remove

Reply:
No i take it back, I'm keeping it.

https://github.com/scottstanie/dolphin/actions/runs/3759878783/jobs/6389954007#step:3:767 

It works on my local build, but on the github runners, the `pip install --no-deps .` step fails. I believe this might be because I'm using the `setuptools_scm` auto versioning, so it needs get to get the hash for the version when it installs.

Reply:
i'm open to alternative fixes to this if we're trying to optimize the Docker build https://github.com/pypa/setuptools_scm/issues/77 but maybe we can do that in a future PR

Reply:
Oh I forgot about `setuptools_scm`. That makes sense that you would need `git` installed in order to create the `_version.py` file.